### ISSUE TITLE
Unknown command: 'bootstrapdemoconfig'

### ISSUE THREAD

**Issue by bibliotechy** [WSULib/combine-playbook#11]
I'm in the vagrant box deployed with Ansible. The web application is working as expected.

I tried to follow [setup the demo configurations[(https://github.com/WSULib/combine/blob/dev/docs/quickstart.md#configuration) to help me understand how they work, but I got the following error.
```
(combine) combine@combine:/opt/combine$ ./manage.py bootstrapdemoconfig
Unknown command: 'bootstrapdemoconfig'
```

**Comment by ghukill** [comment]
Timing is impeccable!  Just noticed the same myself, command got changed to, but I neglected to update the quickstart guide:
```
./manage.py quickstartbootstrap
```

Glad to hear you've giving it a spin!  Let us know if you encounter any other oddities.  One other quick fix that was *just* added to the server provisioning, but absent as of a couple of hours ago, is collecting static files in Django.  Can be done from `/opt/combine`:
```
./manage.py collectstatic
```

This will provide styling and other needed resources for Django admin at `/admin`
